### PR TITLE
Refactor(eos_designs): Schema updates fabric_variable.md

### PR DESCRIPTION
**Pull request by mayurgs34**
## Related Issue(s)

Fixes #<ISSUE ID>

## Component(s) name

`arista.avd.<role-name>`

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/bgp_peer_groups.schema.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@ keys:
             default: IPv4-UNDERLAY-PEERS
           password:
             type: str
+            description: Encrypted Password
```
Comment (Shivani-chourasiya) [inline ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/bgp_peer_groups.schema.yml:20]: there is one more key structured_config in the doc

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/enable_trunk_groups.schema.yml [hunk 2]
```diff
@@ -5,7 +5,6 @@ type: dict
 keys:
   enable_trunk_groups:
     type: bool
-    display_name: Enable Trunk Group support across eos_designs
```
Comment (Shivani-chourasiya) [inline ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/enable_trunk_groups.schema.yml:8]: please add this to description

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/isis_advertise_passive_only.schema.yml [hunk 3]
```diff
@@ -4,5 +4,6 @@
 type: dict
 keys:
   isis_advertise_passive_only:
+    description: ISIS Advertise Passive Only
```
Comment (Shivani-chourasiya) [inline ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/isis_advertise_passive_only.schema.yml]: Please remove this, it is same as Key

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/isis_default_is_type.schema.yml [hunk 4]
```diff
@@ -4,6 +4,7 @@
 type: dict
 keys:
   isis_default_is_type:
+    description: ISIS Default IS Type
```
Comment (Shivani-chourasiya) [inline ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/isis_default_is_type.schema.yml]: Please remove this, it is same as Key

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/isis_ti_lfa.schema.yml [hunk 5]
```diff
@@ -4,6 +4,7 @@
 type: dict
 keys:
   isis_ti_lfa:
+    description: ISIS TI-LFA
```
Comment (Shivani-chourasiya) [inline ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/isis_ti_lfa.schema.yml]: Please remove this, it is same as Key

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/trunk_groups.schema.yml [hunk 6]
```diff
@@ -4,6 +4,7 @@
 type: dict
 keys:
   trunk_groups:
+    description: Trunk Group Names
```
Comment (Shivani-chourasiya) [inline ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/trunk_groups.schema.yml]: Please remove this, it is same as Key

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/underlay_isis_instance_name.schema.yml [hunk 7]
```diff
@@ -4,6 +4,7 @@
 type: dict
 keys:
   underlay_isis_instance_name:
+    description: Underlay ISIS Instance Name
```
Comment (Shivani-chourasiya) [inline ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/underlay_isis_instance_name.schema.yml]: Please remove this, it is same as Key

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/underlay_ospf_area.schema.yml [hunk 8]
```diff
@@ -4,6 +4,7 @@
 type: dict
 keys:
   underlay_ospf_area:
+    description: Underlay OSPF Area
```
Comment (Shivani-chourasiya) [inline ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/underlay_ospf_area.schema.yml]: Please remove this, it is same as Key

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/underlay_ospf_max_lsa.schema.yml [hunk 9]
```diff
@@ -4,6 +4,7 @@
 type: dict
 keys:
   underlay_ospf_max_lsa:
+    description: Underlay OSPF Max LSA
```
Comment (Shivani-chourasiya) [inline ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/underlay_ospf_max_lsa.schema.yml]: Please remove this, it is same as Key

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/bgp_peer_groups.schema.yml (hunk 1 above)
Comment (mayurgs34) [reply ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/bgp_peer_groups.schema.yml:20]: structured_config key added

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/enable_trunk_groups.schema.yml (hunk 2 above)
Comment (mayurgs34) [reply ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/enable_trunk_groups.schema.yml:8]: description added

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/isis_ti_lfa.schema.yml (hunk 5 above)
Comment (mayurgs34) [reply ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/isis_ti_lfa.schema.yml]: Key removed

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/trunk_groups.schema.yml (hunk 6 above)
Comment (mayurgs34) [reply ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/trunk_groups.schema.yml]: Key removed

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/underlay_isis_instance_name.schema.yml (hunk 7 above)
Comment (mayurgs34) [reply ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/underlay_isis_instance_name.schema.yml]: Key removed

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/underlay_ospf_max_lsa.schema.yml (hunk 9 above)
Comment (mayurgs34) [reply ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/underlay_ospf_max_lsa.schema.yml]: Key removed

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/underlay_ospf_area.schema.yml (hunk 8 above)
Comment (mayurgs34) [reply ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/underlay_ospf_area.schema.yml]: Key removed

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/isis_advertise_passive_only.schema.yml (hunk 3 above)
Comment (mayurgs34) [reply ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/isis_advertise_passive_only.schema.yml]: Key removed

**Pull request closed without merging**